src/strategyHelpers/deploymentHelper.ts

```diff
@@ -150,8 +150,15 @@ export async function annotateAndLabelResources(
    resourceTypes: Resource[],
    allPods: any
 ) {
+   const defaultWorkflowFileName = 'k8s-deploy-failed-workflow-annotation'
    const githubToken = core.getInput('token')
-   const workflowFilePath = await getWorkflowFilePath(githubToken)
+   let workflowFilePath
+   try {
+      workflowFilePath = await getWorkflowFilePath(githubToken)
+   } catch (ex) {
```

Comment:
How does this work? I don't see anything in `getWorkflowFilePath` that throws.

Reply:
Makes sense to me. Might be good to just link that issue since it's in theory something broken on GitHub's side and will hopefully be fixed sometime in the future.